src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -307,5 +318,49 @@ public void parseRelationship_validValueWithWhitespace_returnsTrimmedRelationshi
     }
 
 
+    @Test
+    public void parseLocalDateTime_validInput_success() throws Exception {
```

Comment:
tests the different types of inputs, using both the actual dates and also the day of week, for both success and failure